.github/workflows/continuous-integration-workflow.yml

```
@@ -183,7 +196,57 @@ jobs:
 
     - name: Build and publish to GitHub Packages
       uses: docker/build-push-action@v3
```

```suggestion
      uses: docker/build-push-action@v6
```
I updated the version of the action in `speculos-builder` workflow just before we merged yours.